voorhoede/head-start
# fix: Astro & Cloudflare warnings

# Changes

- Adds empty `.dev.vars` to silence Cloudflare dev warnings.
- Prefixes `pages/_*.query.graphql` files with an underscore to silence Astro dev warnings.

# Associated issue

N/A

# How to test

1. Open preview link
2. Verify pages still render
3. Run locally
4. Verify Astro en Cloudflare no longer log warnings

# Checklist

- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
- ~~I have notified a reviewer~~ (discussed with @phortuin)
